src/main/java/mat/client/measurepackage/MeasurePackagePresenter.java

```diff
@@ -168,70 +168,58 @@ public void onClick(ClickEvent event) {
+                clearMessages();
             }
         });
 
 
-        view.getaddRiskAdjVariablesToMeasure().addClickHandler(new ClickHandler() {
-
-            @Override
-            public void onClick(ClickEvent event) {
-                if (MatContext.get().getMeasureLockService().checkForEditPermission()) {
-                    showMeasurePackagerBusy(true);
-                    clearMessages();
-                    updateRiskAdjFromView(currentDetail);
-                    ((Button) view.getPackageMeasureButton()).setEnabled(true);
-                    MatContext.get().getPackageService().saveRiskVariables(currentDetail, new AsyncCallback<Void>() {
-                        @Override
-                        public void onFailure(final Throwable caught) {
-                            logger.log(Level.SEVERE, "Error in PackageService.saveRiskVariables. Error message: " + caught.getMessage(), caught);
-                            if (caught instanceof SaveRiskAdjustmentVariableException) {
-                                getMeasurePackageOverview(MatContext.get().getCurrentMeasureId());
-                                view.getRiskAdjustmentVariableErrorMessageDisplay().createAlert(caught.getLocalizedMessage());
-                            } else {
-                                view.getRiskAdjustmentVariableErrorMessageDisplay().createAlert(MatContext.get().getMessageDelegate().getUnableToProcessMessage());
-                            }
-
-                            showMeasurePackagerBusy(false);
-                        }
-
-                        @Override
-                        public void onSuccess(final Void result) {
+        view.getaddRiskAdjVariablesToMeasure().addClickHandler(event -> {
+            if (MatContext.get().getMeasureLockService().checkForEditPermission()) {
+                showMeasurePackagerBusy(true);
+                clearMessages();
+                updateRiskAdjFromView(currentDetail);
+                ((Button) view.getPackageMeasureButton()).setEnabled(true);
+                MatContext.get().getPackageService().saveRiskVariables(currentDetail, new AsyncCallback<Void>() {
+                    @Override
+                    public void onFailure(final Throwable caught) {
+                        logger.log(Level.SEVERE, "Error in PackageService.saveRiskVariables. Error message: " + caught.getMessage(), caught);
+                        if (caught instanceof SaveRiskAdjustmentVariableException) {
                             getMeasurePackageOverview(MatContext.get().getCurrentMeasureId());
-                            view.getRiskAdjustmentVariableSuccessMessageDisplay().createAlert(MatContext.get().getMessageDelegate().getRiskAdjSavedMessage());
-                            showMeasurePackagerBusy(false);
+                            view.getRiskAdjustmentVariableErrorMessageDisplay().createAlert(caught.getLocalizedMessage());
+                        } else {
```

Comment:
I would suggest adding more logging to onFailure and onSuccess

Reply:
Added couple of Logging statements, I didn’t work on this feature, re-formatted some code.